Java/Vishal/BubbleSort.java

```diff
@@ -0,0 +1,17 @@
+//TimeComplexity = O(n^2)
+//SpaceComplexity = O(1)
+public void bubbleSort() {
+        int[] inputArray = new int[] {12,2,25,1,45,123,85};
+        for(int j=0; j<inputArray.length;j++) {
+            for(int i=0; i<inputArray.length-1; i++){
```

Comment:
every time we don't need to check till the end of the array, as we moved the biggest value to last place